gateway/gateway.go

```diff
@@ -510,7 +510,12 @@ func (gateway *HandleT) userWebRequestWorkerProcess(userWebRequestWorker *userWe
 				anonIDFromReq := strings.TrimSpace(vjson.Get("anonymousId").String())
 				userIDFromReq := strings.TrimSpace(vjson.Get("userId").String())
 				if builtUserID == "" {
```

Comment:
Will there ever be a case when it won't enter into this "if" condition?

Reply:
We initialise `builtUserID` with `""` so this if is always entered

Reply:
Does it makes sense to remove it to make it cleaner?

Reply:
It wont enter the if case when each payload has multiple events and the userId is already built